### PR TITLE
LTO_LOGS needed defined

### PR DESCRIPTION
**Pull request by BleakleyMcD**
LTO_LOGS needed defined to $LTO_INDEX_DIR from mmconfig or logs would attempt to write to /

added in printf to confirm location of LTO_LOGS before script runs

### CODE REVIEW

File: writelto [hunk 1]
```diff
@@ -12,6 +12,8 @@ TAPE_SERIAL_REGEX="^[A-Z0-9]{6}(L[567])?$"
 HIDDEN_FILES=""
 TAPE_EJECT="Y"
 _check_for_lto_index_dir
+LTO_LOGS="${LTO_INDEX_DIR}"
+printf "LTO_LOGS = ${LTO_INDEX_DIR}"
```
Comment (retokromer) [inline writelto]: Why is this needed?

**Comment by BleakleyMcD** [comment]
becuase $LTO_INDEX_DIR is never used writelto. $LTO_LGOS is used instead. 
_check_for_lto_index_dir in mmfunctions only defines $LTO_INDEX_DIR.
in 951b4ad  LTO_LOGS is defined: LTO_LOGS="${HOME}/Documents/lto_indexes"
This got lost in the changes (my fault!) and without it $LTO_LOGS is empty and so paths, such as SCHEMADIR="${LTO_LOGS}/schema", try to write to root, which gets denied and the script exits in error.

File: writelto (hunk 1 above)
Comment (BleakleyMcD) [inline writelto]: its not necessarily, but it lets one know where the logs are going to write to before the script starts running.

Comment (retokromer) [inline writelto]: I’ll check, if we use to print the variables here in `LTOpers` when they are set. My feeling is we do not, but of course I may be wrong. I would personally prefer one unified behaviour through all scripts, because that’s easier for the end user.

cc @dericed

Comment (BleakleyMcD) [inline writelto]: Ok. It could just `echo "LTO log reports will be written to ${LTO_LOGS}"`
I do think it is very helpful to know where those files are written to in case a user is unaware of what mmconfig is set to. 
Alternatively something like:
`printf "rsync error report written to:\n\t%s/%s_writelto.txt" "${WRITELTODIR}" "${TAPE_SERIAL}" printf "readback checksums written to:\n\t%s" "${READBACKOUTPUT}"
printf "after ejecting tape, lto schema file written to:\n\t%s" "${SCHEMA_FILE}"`

could be added at line 121 before `case "${TAPE_EJECT}"`

Comment (retokromer) [inline writelto]: @BleakleyMcD I’m personally still not convinced, but let’s move on! Please replace the `printf` with your first proposal

> `echo "LTO log reports will be written to ${LTO_LOGS}"`

and I’ll merge. Thank you!
